pkg/status/status.go

```diff
@@ -262,6 +270,36 @@ func (s *status) updateStatus(previousStatus *configv1.ClusterOperatorStatus) er
 	return err
 }
 
+// setRelatedObjects populates RelatedObjects in the ClusterOperator.Status.
+// RelatedObjects are consumed by https://github.com/openshift/must-gather
+func (s *status) setRelatedObjects() {
+	objectReferences := []configv1.ObjectReference{
+		// Add the operator's namespace which will result in core resources
+		// being gathered
+		{
+			Resource: "namespace",
```

Comment:
I had to use `namespaces` to satisfy CI - https://github.com/openshift/origin/blob/master/test/extended/operators/clusteroperators.go#L74

Reply:
@aravindhp do you want to `retest`, otherwise I will put an `lgtm`

Reply:
Interesting. Moreover I am not sure if the test is correct as `namespace` should also be valid. @sanchezl please weigh in hear. Does it need to `namespaces`?

Reply:
I originally put `namespace` and then ran into the ci error. Also, looks like all `clusteroperator` resources are using `namespaces`.

Reply:
@aravindhp Yes, the ci test is more restrictive than it should be. The plural name is the "correct" form, but the singular alias (or any other alias) should also work.

Reply:
No worries. I will make this plural for now. You can update the test cases whenever you get the chance.